modules/ROOT/pages/remote-ign.adoc

```diff
@@ -2,17 +2,19 @@
 
 With Ignition, you are not limited to the configuration provided locally to a system and you can retrieve other Ignition configs from a remote source. Those configs will then either replace or be merged into the existing config.
 
+The complete list of supported protocols and related options for remote Ignition files is described in the https://coreos.github.io/ignition/specs/[Ignition specification].
+
 The following examples show how to retrieve an Ignition file from a remote source. They are both set to replace the current configuration with a remote Ignition file.
 
-.Retrieving a remote Ignition file via HTTP
+.Retrieving a remote Ignition file via HTTPS
 [source,yaml]
 ----
 variant: fcos
 version: 1.4.0
 ignition:
   config:
     replace:
-      source: http://example.com/sample.ign
+      source: https://example.com/sample.ign
       verification:
         hash: sha512-e2bb19fdbc3604f511b13d66f4c675f011a63dd967b97e2fe4f5d50bf6cb224e902182221ba0f9dd87c0bb4abcbd2ab428eb7965aa7f177eb5630e7a1793e2e6
```

Comment:
In practice, `verification` isn't used much with HTTPS.  I'd be inclined to use it only in the HTTP example.

Reply:
Removed `verification` from all but the plain HTTP example.